### ISSUE TITLE
Remember previous directory when loading tags template.

### ISSUE THREAD

**Issue by nbeaver** [audacity/audacity#586]
I generally keep a tags.xml file in the directory with the project files. Audacity defaults to looking in `~/.audacity-data`, so every time I load the tags file, I have to navigate to the project folder. It would be nice if Audacity would remember the last directory the tags file was loaded from the way it remembers that last directory used to export to.

Steps to reproduce:

1. "File" -> "Export", OK.
2. In "Edit Metadata Tags" window, click "Template" -> "Load"
3. Observe that location defaults to `~/.audacity-data`.
4. Navigate to the project directory and load the tags file.
5. Export, then repeat steps 1 through 2.

Expected behavior: Audacity defaults to the most recently used tags file.

Observed behavior: Audacity still defaults to `~/.audacity-data`.

Link to relevant part of the code:

    fn = FileNames::SelectFile(FileNames::Operation::_None,
       XO("Save Metadata As:"),
       FileNames::DataDir(),
       wxT("Tags.xml"),
       wxT("xml"),
       { FileNames::XMLFiles },
       wxFD_SAVE | wxFD_OVERWRITE_PROMPT | wxRESIZE_BORDER,
       this);

https://github.com/audacity/audacity/blob/16d52f63a4183bba77ef7305d14622958dc0d1d5/src/Tags.cpp#L1290

Using Audacity version 2.2.1 on Ubuntu 18.04.

**Comment by LWinterberg** [comment]
is this still an issue in 3.1.3?

**Comment by nbeaver** [comment]
Yes, at least I can reproduce it with the snap on Ubuntu 18.04.6.

    $ snap info audacity | grep -A 5 channels
    channels:
      latest/stable:    3.1.3 2022-02-24 (971) 195MB -
      latest/candidate: 3.1.3 2022-02-24 (971) 195MB -
      latest/beta:      ^                            
      latest/edge:      3.1.3 2022-03-03 (973) 195MB -
    installed:          3.1.3            (971) 195MB -

Furthermore, the code is unchanged:

https://github.com/audacity/audacity/blob/cf28e5e1c987cbdfd39a5c37cf4ea35aa4107f59/src/TagsEditor.cpp#L650

**Comment by LWinterberg** [comment]
Let's add another directory preference for the tags/metadata folder and have it empty (most recently used) by default. 

![Image](https://user-images.githubusercontent.com/87814144/172831627-4c91608f-6ada-4ed0-bb46-5e4cacb00752.png)
